docker-for-mac/troubleshoot.md

```diff
@@ -115,6 +115,24 @@ To view the contents of the diagnostic file, run:
 $ open /tmp/BE9AFAAF-F68B-41D0-9D12-84760E6B8740/20190905152051.zip
 ```
 
+### Experimental self-diagnose tool
+
+Docker Desktop 3.6.0 contains an experimental "self-diagnose" tool which helps identify some common
+problems. To run it: first locate `com.docker.diagnose`. If you have installed Docker Desktop
+in the Applications directory, then it is located at
+`/Applications/Docker.app/Contents/MacOS/com.docker.diagnose`.
+
+To run the self-diagnose tool:
+
+```console
+$ /Applications/Docker.app/Contents/MacOS/com.docker.diagnose check
+```
+
+The tool will run a suite of checks and display "PASS" or "FAIL" next to each one. If there are failures
+it will try to highlight the most relevant at the end.
+
+Note the tool is still experimental.
```

Comment:
Wondering if we should ask users to provide feedback. We could add something on the lines of  

> **Note**
>
> The Self-diagnose tools is still experimental. Let us know your feedback by creating an issue in the [for-mac ](https://github.com/docker/for-mac/issues) GitHub repository.

Reply:
Makes sense! I've pushed an update -- let me know what you think. (I'm not sure if it should be `#### Note` or something else in this context)

Reply:
Aah, sorry, I should have added it as a suggestion.

Reply:
Thanks, suggestions certainly make things easier :)